app/views/authentication_fallback_mailer/sign_in_fallback.text.erb

```diff
@@ -0,0 +1,5 @@
+# <%= t('hiring_staff.temp_login.email.heading') %>
```

Comment:
Can we please standardise views with using HAML?

Reply:
I don't know how to write a mailer view using haml. I copied this across from Apply.

Reply:
haml vs erb is a contentious thing in our team, so I am defaulting to 'do as little work as necessary'

Reply:
All you need to do in order to use HAML is to rename the file in `sign_in_fallback.text.haml` and use HAML syntax which is quite easy in this case. I would prefer one standard approach, but I'll approve it in ERB too

Reply:
I am changing it to HAML, but I will need to test this out on an environment to find any breaking changes to the email.

Reply:
Tested